### PR TITLE
[candi] Prevent accidental modification of files in /root/.bashrc and /etc/inputrc

### PR DESCRIPTION
**Pull request by AleksZimin**
## Description

Add checks for sed in the 039_add_kubectl_completion.sh script. This will prevent accidental modification of files located in /root/.bashrc and /etc/inputrc.

## Why do we need it, and what problem does it solve?

The /root/.bashrc and /etc/inputrc files were being modified every time bashible was started.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: candi
type: fix
summary: Prevent accidental modification of files in /root/.bashrc and /etc/inputrc
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AleksZimin. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AleksZimin. I've started the workflow for label 'e2e/run/gcp'...

File: candi/bashible/common-steps/all/039_add_kubectl_completion.sh.tpl [hunk 1]
```diff
@@ -12,14 +12,24 @@
...
-sed -i 's/01;32m/01;31m/' /root/.bashrc
+if grep -qF '#force_color_prompt=yes' /root/.bashrc; then
+  sed -i 's/\#force_color_prompt=yes/force_color_prompt=yes/' /root/.bashrc
+fi
+if grep -qF '01;32m' /root/.bashrc; then
+  sed -i 's/01;32m/01;31m/' /root/.bashrc
+fi
 
 mkdir -p /etc/bash_completion.d
 kubectl completion bash >/etc/bash_completion.d/kubectl
```
Comment (RomanenkoDenys) [inline candi/bashible/common-steps/all/039_add_kubectl_completion.sh.tpl:30]: we need to check if /etc/bash_completion.d/kubectl already exists

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
some nits

File: candi/bashible/common-steps/all/039_add_kubectl_completion.sh.tpl (hunk 1 above)
Comment (AleksZimin) [reply candi/bashible/common-steps/all/039_add_kubectl_completion.sh.tpl]: Fixed

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AleksZimin. I've started the workflow for label 'e2e/run/gcp'...

**Review by RomanenkoDenys — APPROVED** [review]

**Pull request merged**